app/assets/stylesheets/views/_homepage.scss

```diff
@@ -1,3 +1,6 @@
+@import "govuk_publishing_components/components/mixins/prefixed-transform";
```

Comment:
Eyyyyy nice 😎  During cleanup we can separate out your fancy grid work in the gem and expose it in the same way.

Reply:
Yeah, I just didn't want putting the mixin in the gem to delay this work. Since it's a mixin it won't add any extra to the compiled CSS, so I didn't think it too bad a compromise.